lua/terrortown/menus/gamemode/equipment.lua

```diff
@@ -1,12 +1,61 @@
 --- @ignore
 
+local tableCopy = table.Copy
+
+local virtualSubmenus = {}
+
 CLGAMEMODEMENU.base = "base_gamemodemenu"
 
 CLGAMEMODEMENU.icon = Material("vgui/ttt/vskin/helpscreen/equipment")
 CLGAMEMODEMENU.title = "menu_equipment_title"
 CLGAMEMODEMENU.description = "menu_equipment_description"
 CLGAMEMODEMENU.priority = 49
 
+CLGAMEMODEMENU.isInitialized = false
+CLGAMEMODEMENU.lang = "en"
```

Comment:
where do you use this?

Reply:
In GetSubmenus to make sure the virtual menus are only created at first call or when the language did change.
Thinking about making that even more effective and would start with lang = nil.